addons/packages/kpack/0.5.0/README.md

```diff
@@ -0,0 +1,565 @@
+   > Note: To use a private git repo follow the instructions in [secrets](https://github.com/pivotal/kpack/blob/main/docs/secrets.md)
+
+   You can now check the status of the image resource.
+
+   ```bash
+   kp image status tutorial-image -n default
+   ```
+
+   You should see that the image resource has a status Building as it is currently building.
+
+    ```
+    Status:         Building
+    Message:        --
+    LatestImage:    --
+    
+    Source
+    Type:        GitUrl
+    Url:         https://github.com/spring-projects/spring-petclinic
+    Revision:    82cb521d636b282340378d80a6307a08e3d4a4c4
+    
+    Builder Ref
+    Name:    base
+    Kind:    Builder
+    
+    Last Successful Build
+    Id:              --
+    Build Reason:    --
+    
+    Last Failed Build
+    Id:              --
+    Build Reason:    --
+    ```
+
+   You can tail the logs for image resource that is currently building using
+   the [kp cli](https://github.com/vmware-tanzu/kpack-cli/blob/main/docs/kp_build_logs.md)
+
+    ```
+    kp build logs tutorial-image -n default
+    ``` 
+
+   Once the image resource finishes building you can get the fully resolved built OCI image with `kubectl get`
+
+    ```bash
+    kp image status tutorial-image -n default
+    ```
+
+   The output should look something like this:
+    ```
+    Status:         Ready
+    Message:        --
+    LatestImage:    index.docker.io/your-project/app@sha256:6744b...
+    
+    Source
+    Type:        GitUrl
+    Url:         https://github.com/spring-projects/spring-petclinic
+    Revision:    82cb521d636b282340378d80a6307a08e3d4a4c4
+    
+    Builder Ref
+    Name:    base
+    Kind:    Builder
+    
+    Last Successful Build
+    Id:              1
+    Build Reason:    BUILDPACK
+    Git Revision:    82cb521d636b282340378d80a6307a08e3d4a4c4
+    
+    BUILDPACK ID                           BUILDPACK VERSION    HOMEPAGE
+    paketo-buildpacks/ca-certificates      2.4.0                https://github.com/paketo-buildpacks/ca-certificates
+    paketo-buildpacks/bellsoft-liberica    8.4.0                https://github.com/paketo-buildpacks/bellsoft-liberica
+    paketo-buildpacks/gradle               5.5.0                https://github.com/paketo-buildpacks/gradle
+    paketo-buildpacks/executable-jar       5.2.0                https://github.com/paketo-buildpacks/executable-jar
+    paketo-buildpacks/apache-tomcat        6.1.0                https://github.com/paketo-buildpacks/apache-tomcat
+    paketo-buildpacks/dist-zip             4.2.0                https://github.com/paketo-buildpacks/dist-zip
+    paketo-buildpacks/spring-boot          4.5.0                https://github.com/paketo-buildpacks/spring-boot
+    
+    Last Failed Build
+    Id:              --
+    Build Reason:    --
+    ```
+
+   The latest built OCI image is available to be used locally via `docker pull` and in a kubernetes deployment.
+
+7. Run the built app locally
```

Comment:
Global comment.  Add period to the end of opening sentence in step. And also to  opening sentences of steps 2,3,4,6,7,8,9 and 2,3,4,5,6,7, 8 .